code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -273,6 +273,8 @@
 //		reagents.add_reagent("nutriment", 2)							//	this line of code for all the contents.
 //		bitesize = 3													//This is the amount each bite consumes.
 
+// Remember to add anything that should be accepted as biomass by the cloning pods into the GLOB.cloner_biomass_items list.
```

Comment:
No need to place comments here in my opinion. The file has nothing to do with cloning itself

Reply:
Fair enough.